### PR TITLE
#167190445 Like or Dislike an Article

### PR DESCRIPTION
**Pull request by Hadeneekeh**
#### What does this PR do?
Have a user can like or dislike an article

#### Description of Task to be completed?
- Setup the Likes table and associate with Users and Articles table
- Write test
- Create `POST api/v1/articles/:articleId/likes`
- Add the feature to the api documentation

#### How should this be manually tested?
- Clone the repo
- Run the command below in your terminal to install all dependencies
```
npm install
``` 
- Hit the like or dislike an article endpoint `POST api/v1/articles/:articleId/likes` as an authorised user
- Make a request as it is done in the screenshots

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#167190445](https://www.pivotaltracker.com/story/show/167190445)

#### Screenshots (if appropriate)

Like an article by sending `true` and dislike by sending `false`
image

Get that article using slug

image

#### Questions:
None

### CODE REVIEW

**Review by cvjude — APPROVED** [review]
LGTM

File: server/controllers/articleController.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 import models from '../db/models';
-import utils from '../helpers/Utilities';
 import Paginate from '../helpers/paginate';
+import Utilites from '../helpers/Utilities';
 
 const { Op } = sequelize;
 const { paginateArticles } = Paginate;
+const { errorStat, successStat } = Utilites;
+
+const parseBool = (string) => {
```
Comment (chuxmykel) [inline server/controllers/articleController.js:13]: 😂🤣
Nice one @Hadeneekeh . I think this should be added to JavaScript

File: server/middlewares/inputValidator.js [hunk 2]
```diff
@@ -105,6 +106,19 @@ class InputValidator {
...
+  /**
+   * @method validateCommentMessage
+   * @description Validate message input made by user
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {function} next - The next function to point to the next middleware
+   * @returns {function} validate() - An execucted validate function
+   */
+  static validateLikes(req, res, next) {
+    const likes = { ...req.body, ...req.params };
```
Comment (chuxmykel) [inline server/middlewares/inputValidator.js:134]: @chiboycalix Check this out

**Review by chuxmykel — APPROVED** [review]
LGTM

File: server/controllers/articleController.js (hunk 1 above)
Comment (chiboycalix) [reply server/controllers/articleController.js:13]: Hahahahaha

File: server/middlewares/inputValidator.js (hunk 2 above)
Comment (chiboycalix) [reply server/middlewares/inputValidator.js:134]: Okay.

**Review by chiboycalix — APPROVED** [review]
LGTM

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by Lundii — APPROVED** [review]
LGTM

**Review by topseySuave — APPROVED** [review]

**Pull request merged**